.github/workflows/rubyonrails.yml

```diff
@@ -40,20 +40,20 @@ jobs:
       - name: Run tests
         run: bin/rake
 
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
```

Comment:
Why this very specific revision of the action?

Reply:
From https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-ruby

> GitHub recommends pinning actions to a commit SHA.
> To get a newer version, you will need to update the SHA.
> You can also reference a tag or branch, but the action may change without warning.

The positive view of it is its for your own good / safety. The cynical view is it reduces support requests on GitHub. :-)